test/api/test_catalog.py

```diff
@@ -253,19 +266,30 @@ def assert_metadata_record(schema_id):
         expected_metadata = metadata_examples[schema_id]
         if schema_id == 'SchemaOrg.Dataset':
             expected_metadata['identifier'] = f'doi:{example_record.doi}' if example_record.doi else None
-            if has_iso19115:
-                expected_metadata['keywords'] = [
-                    dk['keyword'] for dk in metadata_examples['SAEON.ISO19115']['descriptiveKeywords']
-                    if dk['keywordType'] in ('general', 'place', 'stratum')
-                ]
-            else:
-                expected_metadata['keywords'] = [
-                    s['subject'] for s in metadata_examples['SAEON.DataCite4']['subjects']
-                ]
+            expected_metadata['keywords'] = iso_keywords if has_iso19115 else generic_keywords
             expected_metadata['@id'] = expected_metadata['url'] = (
                 'http://odp.catalog/mims/'
                 f'{example_record.doi if example_record.doi else example_record.id}'
             )
+        elif schema_id == 'RIS.Citation':
+            expected_metadata['ris'] = expected_metadata['ris'].split('\n')
+            metadata_record['metadata']['ris'] = metadata_record['metadata']['ris'].split('\n')
+
+            test_doi = "DO - 10.5555/Test-100"
+            for item in metadata_record['metadata']['ris']:
+                if "DO -" in item:
+                    metadata_record['metadata']['ris'].remove(item)
+                    break
+
+            metadata_record['metadata']['ris'].append(test_doi)
```

Comment:
I think the `DO` tag in the expected metadata could be modified here to match the example record's DOI (or removed if the example record does not have a DOI), rather than modifying the actual response from the API.